web-ng/src/app/services/data-store/data-store.service.ts

```diff
@@ -26,6 +26,7 @@ import { Layer } from './../../shared/models/layer.model';
 import { List, Map } from 'immutable';
 import { Observation } from '../../shared/models/observation/observation.model';
 import { Role } from '../../shared/models/role.model';
+import * as firebase from 'firebase/app';
```

Comment:
Can we instead import only what's needed here?

Reply:
Done.